signup.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html><head>
```

Comment:
<img width="1419" alt="screen shot 2018-06-19 at 17 57 50" src="https://user-images.githubusercontent.com/13004199/41605560-575e697a-73ea-11e8-99b7-2b04f9d6de42.png">
Thanks for the User interface I know styling isn't easy in the beginning but I would appreciate if you had this form in the center of the page